lib/ruby_lsp/requests/support/syntax_tree_formatting_runner.rb

```diff
@@ -25,8 +25,11 @@ def initialize
             )
         end
 
-        sig { params(_uri: String, document: Document).returns(T.nilable(String)) }
-        def run(_uri, document)
+        sig { params(uri: String, document: Document).returns(T.nilable(String)) }
+        def run(uri, document)
+          relative_path = Pathname.new(URI(uri).path).relative_path_from(Pathname.new(WORKSPACE_URI.path))
```

Comment:
I think the argument to `relative_path_from` can just be a string, we don't need to instantiate another `Pathname`.

Reply:
fixed https://github.com/Shopify/ruby-lsp/pull/670/commits/e52212e8bf728837c6582fd2b885827893a7ede2

First time sorbet inferred `WORKSPACE_URI.path` as `T.nilable(String)` and I avoided error by `Pathname.new`, but workspace path should be present in this context.
In `lib/ruby_lsp/requests/diagnostics.rb` I annotated this as `T.must` and same for here.

Is this proper approach? If you find this approach is forcible, please let me know.